userspace/libsinsp/container.cpp

```diff
@@ -763,6 +815,40 @@ bool sinsp_container_manager::parse_docker(sinsp_container_info* container)
 		container->m_imageid = imgstr.substr(cpos + 1);
 	}
 
+	sinsp_utils::split_container_image(container->m_image,
+					   container->m_imagerepo,
+					   container->m_imagetag,
+					   container->m_imagedigest);
+
+	if(container->m_imagedigest.empty())
+	{
+		string img_json;
+#ifndef CYGWING_AGENT
+		if(get_docker("http://dummy/" + m_api_version + "/images/" + container->m_imageid + "/json?digests=1", img_json) == sinsp_docker_response::RESP_OK)
+#else
+		if(get_docker("GET /v1.30/images/" + container->m_imageid + "/json?digests=1 HTTP/1.1\r\nHost: docker \r\n\r\n", img_json) == sinsp_docker_response::RESP_OK)
+#endif
+		{
+			size_t img_pos = img_json.find("{");
```

Comment:
Given that we removed a similar search for '{' above, can we also remove it here?

Reply:
Yes I missed to remove it after a refactoring, it's not needed